code/modules/paperwork/folders.dm

```diff
@@ -67,7 +67,15 @@
 /obj/item/folder/update_overlays()
 	. = ..()
 	if(contents.len)
-		. += "folder_paper"
+		var/to_add = get_paper_overlay()
+		if (to_add)
+			. += to_add
+
+/obj/item/folder/proc/get_paper_overlay()
+	var/mutable_appearance/paper_overlay = mutable_appearance(icon, "folder_paper", offset_spokesman = src)
+	paper_overlay.appearance_flags |= KEEP_APART
```

Comment:
believe the `mutable_appearance` helper already handles setting `appearance_flags`
https://github.com/tgstation/tgstation/blob/410192e9fe62a952462e757b470f4334df29058c/code/datums/mutable_appearance.dm#L26-L32